ra_aid/proc/interactive.py

```
@@ -1,90 +1,179 @@
+        'CI': 'true',
+        'LANG': 'C.UTF-8',
+        'LC_ALL': 'C.UTF-8',
+        'COLUMNS': str(cols),
+        'LINES': str(rows),
+        'FORCE_COLOR': '1',
+        'GIT_TERMINAL_PROMPT': '0',
+        'PYTHONDONTWRITEBYTECODE': '1',
+        'NODE_OPTIONS': '--unhandled-rejections=strict'
+    })
+
+    proc = subprocess.Popen(
```

Ruff (0.8.2)

`subprocess` call: check for execution of untrusted input

(S603)